### PR TITLE
I-12839: Error or Warning needed when using Current Address box with AK or HI State for Pickup Location

### PR DESCRIPTION
**Pull request by TevinAdams**
## Summary

This is to address the issue.  No functionality change but adding additional information as a warning to let users know they will not be able to move forward if AK or HI is selected as their state. 

## Screenshots
image

### CODE REVIEW

File: src/components/Office/ShipmentForm/ShipmentForm.jsx [hunk 1]
```diff
@@ -581,13 +581,17 @@ const ShipmentForm = (props) => {
...
+              not be able to move forward.
+            </Alert>
 
             <div className={styles.ShipmentForm}>
               <div className={styles.headerWrapper}>
                 <div>
                   <ShipmentTag shipmentType={shipmentType} shipmentNumber={shipmentNumber} />
 
-                  <h1>{isCreatePage ? 'Add' : 'Edit'} shipment details</h1>
+                  <h1>{isCreatePage ? 'Add' : 'Edit'} shipment details 3</h1>
```
Comment (danieljordan-caci) [inline src/components/Office/ShipmentForm/ShipmentForm.jsx:594]: `shipment details 3`? What the 3 for?

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]
Need a test or two plz

File: src/components/Office/ShipmentForm/ShipmentForm.jsx (hunk 1 above)
Comment (TevinAdams) [reply src/components/Office/ShipmentForm/ShipmentForm.jsx]: @danieljordan-caci  Sorry removed this. Did not want this change.

**Comment by TevinAdams** [comment]
> Need a test or two plz
@danieljordan-caci  Added tests and corrected some. They were failing.

**Review by danieljordan-caci — APPROVED** [review]

**Pull request merged**